app-frontend/src/app/components/admin/sidebarTeamList/sidebarTeamList.js

```diff
@@ -1,13 +1,55 @@
 import tpl from './sidebarTeamList.html';
 
 const SidebarTeamListComponent = {
+    controller: 'SidebarTeamListController',
     bindings: {
         paginatedResponse: '<',
+        showOrgLogo: '<?',
         sref: '@'
     },
     templateUrl: tpl
 };
 
-export default angular
-    .module('components.sidebarTeamList', [])
-    .component('rfSidebarTeamList', SidebarTeamListComponent);
+const displayLimit = 3;
+
+class SidebarTeamListController {
+    constructor(organizationService) {
+        'ngInject';
+        this.organizationService = organizationService;
+        this.displayLimit = displayLimit;
+    }
+
+    $onInit() {
+        if (this.showOrgLogo) {
+            this.diplayTeams = this.paginatedResponse.results.length <= this.displayLimit ?
+                this.paginatedResponse.results : this.paginatedResponse.results.slice(2);
```

Comment:
I think the slice method here should take something like `this.displayLimit - 1`.

I'm going to suggest a structure change here. I think it would make sense to have a `defaultDisplayLimit` of 3 and a component parameter, `display-limit` that would allow it to be supplied from any code that uses it.

Additionally, I would prefer this portion of the initialization logic to happen unconditionally so that `this.displayTeams` is always defined. I would then change any references in the template to `$ctrl.displayTeam` instead of `$ctrl.paginatedResponse` (it would also allow us to remove the template filter).